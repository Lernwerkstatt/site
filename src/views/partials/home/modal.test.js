const modal = require("./modal.hbs");

describe("modal.hbs", () => {
  it("should match the snapshot", () => {
    expect(modal({})).toMatchSnapshot();
  });
});
