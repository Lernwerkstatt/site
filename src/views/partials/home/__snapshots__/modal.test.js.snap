// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`modal.hbs should match the snapshot 1`] = `
"<div class=\\"modal fade\\" id=\\"\\" tabindex=\\"-1\\" role=\\"dialog\\" aria-labelledby=\\"\\" aria-hidden=\\"true\\">
  <div class=\\"modal-dialog modal-dialog-scrollable\\" role=\\"document\\">
    <div class=\\"modal-content\\">
      <div class=\\"modal-header\\">
        <a href=\\"\\" target=\\"_blank\\">
          <img class=\\"mw-100\\" src=\\"\\" alt=\\"\\">
        </a>
      </div>
      <div class=\\"modal-body text-dark\\">
        <h4 class=\\"mb-3\\" id=\\"\\">
          
        </h4>
        <p>
           Uhr
        </p>
        <p>
          
        </p>
      </div>
      <div class=\\"modal-footer\\">
        <button type=\\"button\\" class=\\"btn btn-secondary\\"><a class=\\"buttonlink\\" href=\\"\\" target=\\"_blank\\">
            Mehr Infos auf Facebook</a></button>
        <button type=\\"button\\" class=\\"btn btn-secondary\\" data-dismiss=\\"modal\\">Schlie√üen</button>
      </div>
    </div>
  </div>
</div>
"
`;
